### PR TITLE
C#: add auto formatting support

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@SaschaMann Don't worry about the actual contents, but if you could look at it from just the GH actions viewpoint, that would be great.

File: languages/csharp/reference/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -69,10 +69,17 @@ Skip this step if you're not sure what to do.
 
 ## Step 5: format code
 
-To format the exercise's code, follow these steps:
+All C# code should be formatted using the [`dotnet format` tool][dotnet-format]. There are three ways to format your C# code:
```
Comment (SaschaMann) [inline languages/csharp/reference/implementing-a-concept-exercise.md:72]: > There are **three** ways to format your C# code:

There are only two listed here

File: languages/csharp/reference/out-of-scope.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The document should answer the question
 The material should be capable of being communicated to an enquiring
 student as and when required.
 
-Where material is covered under "What not to teach" in _design.md_
+Where material is covered under "Out of scope" in _design.md_
```
Comment (SaschaMann) [inline languages/csharp/reference/out-of-scope.md:13]: Should this be in here or did you commit it by accident?

Comment (ErikSchierboom) [reply languages/csharp/reference/out-of-scope.md]: Nope, this should not be in here :)

File: languages/csharp/reference/implementing-a-concept-exercise.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/csharp/reference/implementing-a-concept-exercise.md]: Fixed!

**Comment by SaschaMann** [comment]
LGTM now!

(I hate how rebasing runs CI triggered by commits on master 😑)

**Comment by ErikSchierboom** [comment]
Thanks!

**Pull request merged**